integrations-and-supported-tools/pytorch/notebooks/Neptune_PyTorch.ipynb

```diff
@@ -0,0 +1,467 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Neptune + PyTorch\n",
+    "\n",
+    "Introduction\n",
+    "\n",
+    "This guide will show you how to:\n",
+    "\n",
+    "* Create a NeptuneLogger()\n",
+    "* Log training metrics to Neptune using NeptuneLogger()\n",
+    "* Upload model checkpoints to Neptune using NeptuneLogger()\n",
+    "* Log model predictions to Neptune using NeptuneLogger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FUaDrmRu-h3k"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "* If you're running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example logged to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_0KXqlePqNi"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i36OvZAgvxkj"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "! pip install neptune neptune-pytorch torch torchvision numpy torchviz\n"
```

Comment:
Will replace `neptune neptune-pytorch` with `neptune[pytorch]` once neptune 1.1.2 (or next version) is released.